tests/test_build.py

```diff
@@ -0,0 +1,32 @@
+import os
```

Comment:
Might make sense to write a docstring which gives a high-level overview of the test strategy in here.